Include --pause-isolates-on-start in Dart DevTools (#2475)
dart-lang/site-www: src/tools/dart-devtools.md

### 3. Start the target app

Use the `dart --observe` command to execute the main file

> for the Dart command-line app that you want to debug or observe. Optionally, you can also include the `--pause-isolates-on-start` option which will automatically break the execution at start of the script:

Reviewer: ```suggestion
for the Dart command-line app that you want to debug or observe.
Optionally add `--pause-isolates-on-start`,
which automatically breaks execution at the start of the script.
```
